Source/SVWidgetsLib/Widgets/DataArrayPathSelectionWidget.cpp

```diff
@@ -271,6 +271,19 @@ bool DataArrayPathSelectionWidget::CheckPathRequirements(AbstractFilter* filter,
   return true;
 }
 
+template <typename T>
+bool vectorContains(const std::vector<std::vector<T>>& container, const std::vector<T>& comparison)
+{
+  for(const auto& value : container)
```

Comment:
- std::find
- This isn't the first of these contains helper methods. Consider moving them to a utilities .hpp
- For all intents and purposes, `std::vector<T>` IS `T`.  You can remove one of the `std::vector`s in container.